### DISCUSSION TITLE
Add 'last-updated' in the secondary_info field.

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#7609] [Dashboard] [answered]
not sure if this https://github.com/home-assistant/frontend/pull/6926 already takes care of my request, but since it is merged, am not allowed to ask there.

Please consider adding `last-updated ` to the options for the [`secondary_info:` field for entities](https://www.home-assistant.io/lovelace/entities/#secondary_info) in Lovelace cards.
thanks

**Comment by iantrich** [comment, marked as answer]
It does not but https://github.com/home-assistant/frontend/pull/7433 does

**Reply by Mariusthvdb** [reply]
Great, thanks!
